src/components/layer.js

```diff
@@ -25,4 +26,9 @@ const Layer = ({ children, path }) => {
   )
 }
 
-export default Layer
+Layer.propTypes = {
+  children: PropTypes.string,
```

Comment:
I don't think that `string` is the best option here. Wouldn't it be better to check whether the `children` prop is `node` or an array of nodes?

Reply:
fixed, now
`Layer.propTypes = {
  children: PropTypes.node,
  path: PropTypes.string,
};`